source/game.person.base.bmx

```diff
@@ -519,17 +522,14 @@ Type TPersonBaseCollection Extends TGameObjectCollection
 	Method Remove:Int(obj:TGameObject)
 		If Super.Remove(obj)
 			For Local index:Int = 0 Until FILTER_COUNT
-				If GetFilteredMap(index).Remove(obj.GetID())
-					GetFilteredList(index).Remove(obj)			
-				EndIf
+				filterRefresh[index] = True
```

Comment:
Statt des Einführens einer neuen Liste, könnte man auch filteredIDMaps mit null befüllen. Das hätte für das Aktualisieren denselben Effekt.

Ich habe noch nicht ganz durchstiegen, was den Fehler verursacht. Eine Vermutung ist, dass beim Löschen eines Eintrags das Löschen aus den Maps nicht funktioniert und dadurch ein Eintrag doppelt drin bleibt (Umstellung von GUID auf ID - löschen nach GUID aber ID stimmt nicht überein). Allerdings sollte Babsi nicht gelöscht werden, da der Eintrag nur einmal in der DB ist...